### ISSUE TITLE
Add Exelix/SysDVR support

### ISSUE THREAD

**Issue by thomleg50** [juharris/switch-remoteplay#20]

**Comment by juharris** [comment]
This is the relevant code right https://github.com/exelix11/SysDVR ?
From my understanding, SysDVR can have a noticeable lag. This still seems interesting but I don't think I'll work on it directly. I see this being supported by adding another option to the client for video input. Right now there's only Mixer.

**Comment by thomleg50** [comment]
Yes is it but lag is almost 0 with 4.0 release

Le lun. 8 juin 2020 à 19:11, Justin D. Harris <notifications@github.com> a
écrit :

**Comment by juharris** [comment]
Oh that's good to hear. Thanks. Looks like Version 4.0 came out on May 25, 2020.

**Comment by thomleg50** [comment]
I may help

**Comment by thomleg50** [comment]
If I'm not wrong sysdvr can output to rtsp

**Comment by juharris** [comment]
Oh, it also looks like your console needs to be hacked to use SysDVR. So I definitely won't be doing this or really encouraging it but others can feel free to implement and test integration into the client to add the video stream.

**Comment by thomleg50** [comment]
Yes it need and it's a good way for peoples with such unit and without
capture card

Le lun. 8 juin 2020 à 21:59, Justin D. Harris <notifications@github.com> a
écrit :

**Comment by juharris** [comment]
Yeah good point. I bought the cheapest one capture card I could find for like 70CAD but I understand not everyone can do that. Also mine is limited, it doesn't "split", it only sends output to my PC and not another screen.

**Comment by thomleg50** [comment]
With sysdvr you get screen on computer and on the switch tablet

Le lun. 8 juin 2020 à 22:17, Justin D. Harris <notifications@github.com> a
écrit :
